pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/segmentgenerationandpush/SegmentGenerationAndPushTaskUtils.java

```diff
@@ -50,7 +49,7 @@ static PinotFS getInputPinotFS(Map<String, String> taskConfigs, URI fileURI)
       return pinotFS;
     }
     // Fallback to use the PinotFS created by Minion Server configs
-    return PinotFSFactory.create(fileURIScheme);
+    return null;
```

Comment:
Should we update the comment 1 line above?

Reply:
Thanks for pointing out! Removed it now.